### DISCUSSION TITLE
Gene expression dynamics over time

### DISCUSSION THREAD

**Discussion by alexandra-valeanu** [aristoteleo/dynamo-release#372] [Q&A]
Dear Dynamo creators,

Thank you for the hard work on creating this tool!

I am interested in seeing some gene expressions over time. So far, using the Forebrain Glutamatergic data, dyn.sample_data.hgForebrainGlutamatergic(), with dyn.pl.kinetic_curves and scatter plot, I obtained:

<img width="940" alt="image" src="https://user-images.githubusercontent.com/57878719/175783764-17742e7d-54ce-495e-8796-e997b771a896.png">

What I want to achieve is something like this:

<img width="213" alt="image" src="https://user-images.githubusercontent.com/57878719/175783803-63dd624d-9a3a-4151-b3b6-4dbd620ba1da.png">

What is your recommendation on the parameters to use in oder to reach the desired results?

**Comment by Xiaojieqiu** [comment]
Thanks for using dynamo. first of all, you need to check whether the velocity and the pseudotime calculated. Can you show me the streamline plot, with cell colored by the `umap_ddhoge_potential`?

**Reply by alexandra-valeanu** [reply]
Thank you for your quick response!

<img width="742" alt="image" src="https://user-images.githubusercontent.com/57878719/175790076-e62a2f64-9387-427b-b83e-26f6b1fb00f5.png">
<img width="743" alt="image" src="https://user-images.githubusercontent.com/57878719/175790596-e00d9ba4-e3a3-4628-8cec-3396ceddf0e3.png">

**Reply by Xiaojieqiu** [reply]
thanks for the figures. Can you please follow this notebook to make your velocity results improved? 

https://github.com/aristoteleo/dynamo-tutorials/blob/master/hgForebrainGlutamatergic.ipynb

You may need to slightly update the code in this tutorial to make it compatible with the latest dynamo.  

I have to also admit that: this hgForebrainGlutamatergic is kind hard to get it right (same as other tools) -- see Lior's recent preprint on RNA velocity. If you are just interested in the expression pattern, try plug in velocyto's velocity result (follow velocyto's tutorial) into dynamo's structure and use dynamo to calculate the pseudotime and other downstream cool differential geometry and LAP, in silico perturbation prediction, etc. 

See this functions to get started with moving the velocity data from velocyto to dynamo and good luck!

**Reply by Xiaojieqiu** [reply]
https://github.com/aristoteleo/dynamo-release/blob/ArmanChanges_May2022/dynamo/tools/velocyto_scvelo.py

**Reply by alexandra-valeanu** [reply]
Thank you for your directions!

Actually, I have no interest in this hgForebrainGlutamatergic dataset. I was following the dynamo notebook tutorial on it to represent what I want from my own data, mouse midbrain development, for which I don't need the velocity transition from velocyto to dynamo. Sorry for making it confusing.

I want to know if it's possible to get with dynamo's functions a 'gene expression over time' plot like this:

<img width="205" alt="image" src="https://user-images.githubusercontent.com/57878719/175920892-a015d6b8-726b-4d6c-91fb-a431af12be8b.png">
